### PR TITLE
acatangiu: retain rank 2

### PR DESCRIPTION
**Pull request by acatangiu**
Retention evidence covering period 2024/04/30 -> 2024/07/30.

[Rendered link](https://github.com/acatangiu/Evidences/blob/retention/evidence/acatangiu/0002-acatangiu-2024-Q2-evidence.md)

### CODE REVIEW

**Review by sandreim — APPROVED** [review]

**Review by andresilva — APPROVED** [review]

**Comment by andresilva** [comment]
s/Evanghelism/Evangelism/

File: evidence/acatangiu/0002-acatangiu-2024-Q2-evidence.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Evidence-0002: Retention at Rank 2
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/07/30)                                                             |
```
Comment (ggwpez) [inline evidence/acatangiu/0002-acatangiu-2024-Q2-evidence.md:5]:
```suggestion
| **Report Date** | 2024/07/30                                                             |
```

I think this is part of the template.

File: evidence/acatangiu/0002-acatangiu-2024-Q2-evidence.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+| **Submitted by**| Adrian Catangiu                                                                             |
+
+
+## Member details
+
+- Matrix username: `@adrian:parity.io`
+- Polkadot address: `121dd6J26VUnBZ8BqLGjANWkEAXSb9mWq1SB7LsS9QNTGFvz`
+- Current rank: `2`
+- Date of initial induction: `06.12.2023`
```
Comment (ggwpez) [inline evidence/acatangiu/0002-acatangiu-2024-Q2-evidence.md:14]:
```suggestion
- Date of initial induction: `2023/06/12`
```
Same format everywhere.

**Review by ggwpez — APPROVED** [review]

**Comment by kianenigma** [comment]
https://collectives.subsquare.io/fellowship/referenda/180

**Pull request merged**